### PR TITLE
[v1.1.2] Longhorn UI use `Extension developer features`

### PR DESCRIPTION
**Pull request by n313893254**
image

### CODE REVIEW

**Review by guangbochen — COMMENTED** [review]
@n313893254 please check the correct path of [v1.1 version](https://github.com/harvester/docs/tree/main/versioned_docs/version-v1.1), also would be better to describe the different option names in different versions.

**Review by guangbochen — APPROVED** [review]
LGTM, and will merge before v1.1.2 release, thanks.

File: docs/troubleshooting/harvester.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Users can generate a support bundle in the Harvester GUI with the following step
 
 _Available as of v1.1.0_
 
-You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must enable the `Developer Tools & Features` first via the Preference page.
+You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must enable the `Extension developer features` first via the Preference page.
```
Comment (vickyhella) [inline docs/troubleshooting/harvester.md:70]:
```suggestion
You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must first go to the `Preferences` page and select `Enable Extension developer features` under `Advanced Features`.

File: i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/harvester.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ title: "Harvester"
 
 _从 v1.1.0 起可用_
 
-你现在可以直接在 `Support` 页面上访问嵌入式 Rancher 和 Longhorn 仪表板，但必须先通过 **Preference** 页面启用 `Developer Tools & Features`。
+你现在可以直接在 `Support` 页面上访问嵌入式 Rancher 和 Longhorn 仪表板，但必须先通过 **Preference** 页面启用 `Extension developer features`。
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/troubleshooting/harvester.md:70]:
```suggestion
你现在可以直接在 `Support` 页面上访问嵌入式 Rancher 和 Longhorn 仪表板，但必须先前往 `Preferences` 页面并选中 `Advanced Features` 下的 `Enable Extension developer features`。
```

File: versioned_docs/version-v1.1/troubleshooting/harvester.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ Users can generate a support bundle in the Harvester GUI with the following step
 
 _Available as of v1.1.0_
 
-You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must enable the `Developer Tools & Features` first via the Preference page.
+You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must enable the `Extension developer features` first via the Preference page. (In Harvester v1.1.0 and v1.1.1, the option name is `Developer Tools & Features`)
```
Comment (vickyhella) [inline versioned_docs/version-v1.1/troubleshooting/harvester.md:70]:
```suggestion
You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must first go to the `Preferences` page and select `Enable Extension developer features` under `Advanced Features` (in Harvester v1.1.0 and v1.1.1, the option name is `Developer Tools & Features`).
```

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.1/troubleshooting/harvester.md [hunk 4]
```diff
@@ -67,7 +67,7 @@ title: "Harvester"
 
 _从 v1.1.0 起可用_
 
-你现在可以直接在 `Support` 页面上访问嵌入式 Rancher 和 Longhorn 仪表板，但必须先通过 **Preference** 页面启用 `Developer Tools & Features`。
+你现在可以直接在 `Support` 页面上访问嵌入式 Rancher 和 Longhorn 仪表板，但必须先通过 **Preference** 页面启用 `Extension developer features`。(在Harvester v1.1.1和v1.1.0中，选项名称为`Developer Tools & Features`)
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.1/troubleshooting/harvester.md:70]:
```suggestion
你现在可以直接在 `Support` 页面上访问嵌入式 Rancher 和 Longhorn 仪表板，但必须先前往 `Preferences` 页面并选中 `Advanced Features` 下的 `Enable Extension developer features`（在 Harvester v1.1.1 和 v1.1.0 中，该选项名称为 `Developer Tools & Features`）。
```

**Review by vickyhella — COMMENTED** [review]
Slightly updated the instructions according to the actual operations in our UI.

**Review by vickyhella — APPROVED** [review]

**Pull request merged**